.github/workflows/main.yml

```diff
@@ -28,57 +28,115 @@ jobs:
           repository: hpi-schul-cloud/e2e-system-tests
           path: e2e-system-tests
 
+      - name: Install 1Password CLI
+        uses: 1password/install-cli-action@v1
+
+      - name: Verify 1Password was Installed
+        run: op --version
+
       - name: Configure 1Password Service Account
         uses: 1password/load-secrets-action/configure@v1
         with:
           service-account-token: ${{ secrets.service-account-token }}
 
-      - name: Setting URL's
-        id: setting_url
+      - name: Fetch JSON files from 1Password
+        run: |
+          instances=(brb dbc nbc)
+          environments=(dev ref)
+
+          for instance in "${instances[@]}"; do
+            for environment in "${environments[@]}"; do
+              vault_name="cy-${environment}-${instance}"
+              op document get cypress-json --vault "$vault_name" --output ${{ github.workspace }}/e2e-system-tests/env_variables/file-${environment}-${instance}.json
+            done
+          done
+
+      - name: Aggregate JSON Data into one File
+        id: read_variables
         run: |
+          instances=(brb dbc nbc)
+          environments=(dev ref)
+
+          cypress_brb=""
+          cypress_default=""
+          cypress_nbc=""
           workflow_name="${{ github.workflow }}"
+
+          check_environment() {
+              local url="$1"
+
+              if [[ $url == *"staging"* || $url == *"schulportal"* ]]; then
+                  echo "ref"
+              else
+                  echo "dev"
+              fi
+          }
+
           if [[ $workflow_name == *"manual"* ]]; then
             echo "This is a manual workflow"
-            echo "CYPRESS_BRB=${{ github.event.inputs.instance1 }}" >> $GITHUB_ENV
-            echo "$CYPRESS_BRB"
-            echo "CYPRESS_NBC=${{ github.event.inputs.instance2 }}" >> $GITHUB_ENV
-            echo "$CYPRESS_NBC"
-            echo "CYPRESS_DEFAULT=${{ github.event.inputs.instance3 }}" >> $GITHUB_ENV
-            echo "$CYPRESS_DEFAULT"
+            cypress_brb="${{ github.event.inputs.instance1 }}"
+            cypress_nbc="${{ github.event.inputs.instance2 }}"
+            cypress_default="${{ github.event.inputs.instance3 }}"
```

Comment:
Could you please rename the `default` here to `dbc`? I know it's used interchangeably across the whole SC, but I prefer using `dbc` as the `default` can be too general and without some narrow context it could be misunderstood.

Reply:
fixed